.travis.yml

```diff
@@ -152,11 +152,29 @@ matrix:
         - DESC="All sevntu checks should be used"
         - CMD="./.ci/travis/travis.sh all-sevntu-checks"
 
-    # MacOS verify (till cache is not working, we can not do verify)
+    # MacOS JDK8 verify (till cache is not working, we can not do verify)
     - os: osx
       osx_image: xcode9.3
       env:
-        - DESC="MacOS verify, site, assembly
+        - DESC="MacOS JDK8 verify, site, assembly
+        - CMD1="mvn -e package -Dlinkcheck.skip=true"
+        - CMD2="mvn -e package -Passembly"
+        - CMD="export JAVA_HOME=$(/usr/libexec/java_home) && $CMD1 && $CMD2"
+
+    # MacOS JDK10 verify (till cache is not working, we can not do verify)
+    - os: osx
+      osx_image: xcode10
+      env:
+        - DESC="MacOS JDK10 verify, site, assembly
```

Comment:
lets remove this build.
Reason: MacOS is not very quick item, becaue macos resources on Travis are more expensive, so queue for such resource for free is very long.
I think jdk8 and jdk11 is enough.

Note: it is ok to stay builds for openjdk9|10, it will be enough for testing for non long support jdk.